.github/workflows/test.yml

```diff
@@ -10,31 +10,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # Constrain to single version to simplify install from wheels
-        os: ['ubuntu-latest']
-        python: ['3.7']
-        # os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
-        # python: ['3.7', '3.9', '3.10']
+         os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
+         python: ['3.7', '3.9', '3.10']
```

Comment:
[Consider] We might want to add 3.11 to this now as well.

Reply:
Added in 9fb9a555